### PR TITLE
fixes autocomplete on homepage typeahaed

### PR DESCRIPTION
**Pull request by esotrope**

### CODE REVIEW

File: webapp/src/main/webapp/src/app/shared/form/sb-typeahead.component.ts [hunk 1]
```diff
@@ -15,13 +15,13 @@ import { byString, join } from "@kourge/ordering/comparator";
            [disabled]="disabledInternal"
            [typeahead]="options"
            [typeaheadMinLength]="0"
-           typeaheadWaitMs="300"
```
Comment (esotrope) [inline webapp/src/main/webapp/src/app/shared/form/sb-typeahead.component.ts]: this made the typeahead wait 300ms to display after clicking even though the data was in-hand : /
(this component is currently only used on the home page and school results page where the schools are in-hand after page load)

File: webapp/src/main/webapp/src/app/shared/form/sb-typeahead.component.ts [hunk 2]
```diff
@@ -15,13 +15,13 @@ import { byString, join } from "@kourge/ordering/comparator";
...
            typeaheadOptionField="label"
            typeaheadGroupField="group"
            (typeaheadOnSelect)="onSelectInternal($event.item)"
            (ngModelChange)="onChangeInternal()"
            [(ngModel)]="search"
-           [placeholder]="placeholder">
+           placeholder="{{placeholder}}"
+           autocomplete="off">
```
Comment (esotrope) [inline webapp/src/main/webapp/src/app/shared/form/sb-typeahead.component.ts]: the fix

File: webapp/src/main/webapp/src/app/shared/form/sb-typeahead.component.ts (hunk 1 above)
Comment (wtritch) [reply webapp/src/main/webapp/src/app/shared/form/sb-typeahead.component.ts]: I guess we'll need to put that back once we connect it to a back-end query? =)

File: webapp/src/main/webapp/src/app/shared/form/sb-typeahead.component.ts (hunk 2 above)
Comment (wtritch) [reply webapp/src/main/webapp/src/app/shared/form/sb-typeahead.component.ts]: I wonder if we want to add autocomplete="off" to all of our inputs... I don't suppose we could apply it globally with a CSS selector? (Without using a DOM accessor like jQuery)

**Pull request merged**